mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -76,33 +94,71 @@ CLASS net/minecraft/class_238 net/minecraft/util/math/Box
 	METHOD method_19316 from (Lnet/minecraft/class_3341;)Lnet/minecraft/class_238;
 		ARG 0 mutable
 	METHOD method_29968 from (Lnet/minecraft/class_243;)Lnet/minecraft/class_238;
-		ARG 0 vec3d
+		ARG 0 pos
 	METHOD method_30048 of (Lnet/minecraft/class_243;DDD)Lnet/minecraft/class_238;
 		ARG 0 center
 		ARG 1 dx
 		ARG 3 dy
 		ARG 5 dz
+	METHOD method_35574 withMinX (D)Lnet/minecraft/class_238;
+		COMMENT Creates a box with the minimum X provided and all other coordinates
+		COMMENT of this box.
+		ARG 1 minX
+	METHOD method_35575 withMinY (D)Lnet/minecraft/class_238;
+		COMMENT Creates a box with the minimum Y provided and all other coordinates
+		COMMENT of this box.
+		ARG 1 minY
+	METHOD method_35576 withMinZ (D)Lnet/minecraft/class_238;
+		COMMENT Creates a box with the minimum Z provided and all other coordinates
+		COMMENT of this box.
+		ARG 1 minZ
+	METHOD method_35577 withMaxX (D)Lnet/minecraft/class_238;
+		COMMENT Creates a box with the maximum X provided and all other coordinates
+		COMMENT of this box.
+		ARG 1 maxX
+	METHOD method_35578 withMaxY (D)Lnet/minecraft/class_238;
+		COMMENT Creates a box with the maximum Y provided and all other coordinates
+		COMMENT of this box.
+		ARG 1 maxY
+	METHOD method_35579 withMaxZ (D)Lnet/minecraft/class_238;
+		COMMENT Creates a box with the maximum Z provided and all other coordinates
+		COMMENT of this box.
+		ARG 1 maxZ
+	METHOD method_35580 (DDD)Lnet/minecraft/class_238;
```

Comment:
Should name this one. This one is the opposite of expand, like how shrink is the opposite of stretch.

Stretch/shrink only affects one side (min side if input <0, max side if input >0), while expand/this one (maybe name this `contract`? dunno) affects both min and max.

Reply:
I also prefer adding javadoc link between expand/contract and shrink/stretch to indicate their relations.